src/js/vendor/highlight.bundle.js

```diff
@@ -26,6 +26,7 @@
   hljs.registerLanguage('scala', require('highlight.js/lib/languages/scala'))
   hljs.registerLanguage('shell', require('highlight.js/lib/languages/shell'))
   hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'))
+  hljs.registerLanguage('sqlpp', require('highlight.js/lib/languages/sqlpp'))
```

Comment:
Can we register the alias as `sql++`? Then authors can specify `[code,sql++]` and the file type will appear in the output as `SQL++`.

It seems perfectly fine to have ++ in the language name: see #149

Reply:
c++ is registered as cpp so it might not work for highlight.js.

I'll give it a bash though; might be okay.

Reply:
No it works, I've tried it, see #149

Reply:
🙋‍♂️ my interpretation of what was going on here was wrong. The ++ part of the name is discarded